src/common/motech-list/list.controller.spec.js

```diff
@@ -0,0 +1,26 @@
+describe('List Controller', function () {
```

Comment:
@piopawel Please fix tabulation in your files - it should be four spaces instead of two